### PR TITLE
docs: docs/api/index.md: specify or correct some details

### PR DESCRIPTION
**Pull request by pauamma**
- email address for help if reaching limits

- example contact email address in user agent (myapp.com is registered)

- limit for concurrent sessions from a single IP address

While there, copyedit throughout.

### CODE REVIEW

File: docs/api/index.md [hunk 1]
```diff
@@ -40,11 +40,13 @@ To protect our infrastructure, we enforce rate-limits on the API and the website
 - 100 req/min for all read product queries (`GET /api/v*/product` requests or product page). There is no limit on product write queries.
 - 10 req/min for all search queries (`GET /api/v*/search` or `GET /cgi/search.pl` requests)
 
-If these limits are reached, we reserve the right to deny you the access to the website and the API through IP address ban. If your IP has been banned, feel free to send us an email to explain us the reason why you reached the limits: reverting the ban is possible.
+If these limits are reached, we reserve the right to deny you access to the website and the API through IP address ban. If your IP has been banned, feel free to [email us to explain why you reached the limits][why_reached_limits]: reverting the ban is possible.
 
-If your requests come from your users directly (ex: mobile app), the rate limits will apply per user.
+[why_reached_limits]: mailto:contact@openfoodfacts.org
```
Comment (teolemon) [inline docs/api/index.md:45]:
```suggestion
[why_reached_limits]: mailto:reuse@openfoodfacts.org
```

Comment (pauamma) [reply docs/api/index.md:45]: I can do that ("contact@" was OK'd by @alexgarel). Anything else so I can batch changes in one commit?

Comment (pauamma) [reply docs/api/index.md]: Done. Let me know if you need me to rebase and or squash.

**Review by teolemon — APPROVED** [review]
Sorry for the delay. The PR is perfect 👍 We're putting more emphasis on helping reusers level up with the recently merged related PRs, and every step that eventually gets us fresh data is really impactful. https://github.com/openfoodfacts/openfoodfacts-server/pulls?q=is%3Apr+is%3Aclosed+label%3A%22dev+journeys+documentation%22

**Comment by teolemon** [comment]
thanks @pauamma , merged 👍

**Pull request merged**